lib/rubocop/cop/naming/memoized_instance_variable_name.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Naming
+      # This cop checks for memoized methods whose instance variable name
+      # does not match the method name.
+      #
+      # @example
+      #   # bad
+      #
```

Comment:
Really small nit - No extra lines between `# bad` and `# good` and the actual examples.  See